packages/authorizer/test/AuthorizedHelpers.test.ts

```diff
@@ -0,0 +1,63 @@
+import { deploy, ONES_ADDRESS } from '@mimic-fi/v3-helpers'
+import { expect } from 'chai'
+import { Contract } from 'ethers'
+
+/* eslint-disable no-secrets/no-secrets */
+
+describe('AuthorizedHelpers', () => {
+  let authorizedHelpers: Contract
+
+  before('create authorizer', async () => {
+    authorizedHelpers = await deploy('AuthorizedHelpersMock', [])
+    console.log('f', authorizedHelpers.functions)
+  })
+
+  describe('authParams', () => {
+    const address = ONES_ADDRESS
+    const bytes32 = '0x0000000000000000000000000000000000000000000000000000000000000002'
+    const uint256 = BigInt(100)
```

Comment:
Let's use big numbers or simply numbers 

```suggestion
    const uint256 = bn(100)
```